### DISCUSSION TITLE
增加快捷键

### DISCUSSION THREAD

**Discussion by zhaoo77** [Richasy/Bili.Copilot#209] [Ideas]
大佬大佬，可以加一些基础的快捷键吗，就像网页端那样的，d关字幕，f全屏，等等

**Comment by simuleite** [comment]
非常需要展开详情（以及详情、评论列切换）、刷新推荐视频的快捷键

**Comment by cangming99** [comment]
已经习惯按Q点赞了，全屏看的时候能顺手点赞，不用退出全屏再点赞，希望能添加快捷键
